module/zfs/dsl_deadlist.c

```diff
@@ -112,6 +112,7 @@ unsigned long zfs_livelist_max_entries = 500000;
  */
 int zfs_livelist_min_percent_shared = 75;
 
+boolean_t dsl_deadlist_prefetch_bpobj = B_TRUE;
```

Comment:
Shall we switch this to an `int` so it can be made available as a module option?

Reply:
TBH, I'd rather remove this tunable and make it always prefetch the bpobj's dnodes.  @pzakha, what do you think?

Reply:
I'm fine with that. I don't think we have had situations where we needed to turn that off.